include/optimization/cutting_plane.h

```diff
@@ -0,0 +1,1049 @@
+//
+// Created by panagiotis on 24/5/2019.
+//
+
+#ifndef VOLESTI_CUTTING_PLANE_H
+#define VOLESTI_CUTTING_PLANE_H
+
+
+#include "polytopes.h"
+#include "Eigen"
+#include <list>
+#include <vector>
+#include <unsupported/Eigen/MatrixFunctions>
+#include "samplers.h"
+#include "interior_point.h"
+#include <queue>
+
+int STEPS;
+
+namespace optimization {
+
+
+    const double ZERO = 0.000000000001;
```

Comment:
is this for some numerical stability reason? if so please do not call it ZERO, it is not ;) maybe `epsilon` or `approximation_error` ?